rti/mosaic-rti-api/src/main/java/org/eclipse/mosaic/rti/UNITS.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.rti;
+
+public final class UNITS {
```

Comment:
While I do see the upside of adding these constants, I feel like this name is too generic. All time and also data units could also fit under this umbrella. It would probably also be cleaner to tackle this in a separate PR and MR and replace all code occurrences of conversions that we can find.